jwala-services/src/main/java/com/cerner/jwala/service/app/impl/ApplicationServiceImpl.java

```diff
@@ -139,7 +139,7 @@ public Application updateApplication(UpdateApplicationRequest updateApplicationR
 
     private void updateApplicationWarMetaData(UpdateApplicationRequest updateApplicationRequest, Application application) {
         final String appWarName = application.getWarName();
-        if (!appWarName.isEmpty()) {
+        if (appWarName!=null && !appWarName.isEmpty()) {
```

Comment:
Can we do this instead ?

    if (!StringUtils.isEmpty(appWarName)) {
        ...
    }